var/spack/packages/llvm/package.py

```diff
@@ -41,13 +41,31 @@ class Llvm(Package):
 
     depends_on('python@2.7:')
 
+    variant('libcxx', default=False, description="Builds the LLVM Standard C++ library targeting C++11")
+
+    ##########
+    # @3.7.0
+    resource(name='compiler-rt',
+             url='http://llvm.org/releases/3.7.0/compiler-rt-3.7.0.src.tar.xz', md5='383c10affd513026f08936b5525523f5',
```

Comment:
FYI: It's on my todo list to allow referencing the package's own version in URLs, patch paths, etc.  So you could write something like this:
```
http://llvm.org/releases/${spec.version}/compiler-rt-${spec.version}.src.tar.xz
```
I need to think about how to implement it but it shouldn't be so hard.  I just want to get the syntax right.  Not sure `${}` is the best way to substitute it, either.